apps/ideaspace/src/components/common/IdeaCard/IdeaCard.js

```diff
@@ -1,151 +1,116 @@
-            style={{ width: "100%", height: "100%", borderRadius: "1rem", }}
-          />
-
-        </ImgButton>
-      </Link>
-
-      <atoms.Box flexDirection='column' alignItems='flex-start' justifyContent='space-between'
-        margin='-1.5rem 2rem 1.5rem' style={{ maxWidth: '18.5rem' }}>
+      <atoms.Box>
+        <IdeaCardTag
+          status={tagContent}
+        />
+      </atoms.Box>
 
-        <StatuBox style={{ background: tagColor, }}>{tag}</StatuBox>
+      <IdeaCardImg
+        cardId={cards.id}
+        cardImg={cards.imgSrc}
+      />
 
-        <atoms.Typography type='h3' style={{ fontSize: '1.5rem', textAlign: 'left' }}>
-          {cards.ideaName}
-        </atoms.Typography>
+      <Link href={{ pathname: urlPath }}>
+        <atoms.Box flexDirection='column' alignItems='flex-start' justifyContent='space-between'
+          padding='0rem 2rem 2rem' style={{ maxWidth: '18.5rem' }}>
 
-        {cards.comments.length > 0 ? (
-          <atoms.Box alignItems='center' style={{ marginTop: '-1rem' }} >
-            <img alt='commentSvg' src={commentSvg} />
-            <atoms.Typography type='p' style={{ fontSize: '1rem', textAlign: 'left' }}>
+          <atoms.Typography type='h3' style={{ fontSize: '1.5rem', marginBottom: '2rem' }}>
+            {cards.ideaName}
+          </atoms.Typography>
 
-              &nbsp;COMMENTS:&nbsp;{cards.comments.length}
-            </atoms.Typography>
-          </atoms.Box>
-        ) : (
-          null
-        )}
+          <IdeaCardComment
+            commentLength={cards.comments.length}
+          />
 
-        <atoms.Box alignItems='center' style={{ marginTop: '0.5rem' }} >
-          <img alt='timeSvg' src={timeSvg} />
-          <atoms.Typography type='p' style={{ fontSize: '1rem', textAlign: 'left' }}>
-            &nbsp;UPDATED:&nbsp;{updated}
-          </atoms.Typography>
+          <IdeaCardUpdated
+            updatedAt={cards.updated_at}
+          />
         </atoms.Box>
+      </Link>
 
-        {cardType == "mine" ? (
-          <Link href={{ pathname: `/ideaspace/edit/${cards.id}` }}>
-            <atoms.Button
-              buttonSize='standard'
-              buttonType='alternative'
-              style={{ width: "100%", marginTop: "2rem" }}
-            >
-              WORKSHOP&nbsp;THIS&nbsp;IDEA
-            </atoms.Button>
-          </Link>
-        ) : (
-          <Link href={{ pathname: `/ideaspace/workshop/${cards.id}` }}>
-            <atoms.Button
-              buttonSize='standard'
-              buttonType='alternative'
-              style={{ width: "100%", marginTop: "2rem" }}
-            >
-              HELP&nbsp;THIS&nbsp;IDEA
-            </atoms.Button>
-          </Link>
-        )}
-
-      </atoms.Box>
+      {tagContent == "archived" ? (
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='alternative'
+          style={{ margin: '0rem 2rem 1.5rem' }}
```

Comment:
Let's be mindful of how much inline styles we are adding to the project. This leads to a lot of duplicated code. E.g. we have the same style definition on line 107 of this file.